src/main/resources/configs/ios.properties

```diff
@@ -0,0 +1,4 @@
+deviceName=Iphone 12
```

Comment:
Instead of ios properties i would name it iosDevice properties or smth like that. Ios properties are static and never change while device properties change. Keep in mind you might have parallel execution against different devices

Reply:
Same for android